version.go

```diff
@@ -27,7 +27,7 @@ package gocql
 import "runtime/debug"
 
 const (
-	mainModule = "github.com/gocql/gocql"
+	mainModule = "github.com/apache/cassandra-gocql-driver"
```

Comment:
Have you tested this? It's using this string to match the dependency path but I'm not sure where the dependency path comes from. If it comes from the actual module name then I don't think this will work.

Reply:
```
if d.Path == mainModule {
  driverName = "github.com/apache/cassandra-gocql-driver"
  driverVersion = d.Version
  if d.Replace != nil {
	  driverName = d.Replace.Path
	  driverVersion = d.Replace.Version
  }
break
}
```

Reply:
🤦 No, really... this is what I get for rushing.  Completely agree that "mainModule" is entirely the wrong name here although I still think that there's merit in encapsulating that value in a constant, albeit a better-named constant.

I'll tweak this a bit and see if I can come up with something that (a) is clearer and (b) is very much in the spirit of what's proposed above.

Reply:
Perhaps we could rename `mainModule` to `upstreamModule` ?

Reply:
How about just "mainPacakge"?  That has the advantage of matching the terminology in the [runtime/debug docs](https://pkg.go.dev/runtime/debug#BuildInfo)... ?

Reply:
package in the Go land means somthing different than module, in this case it's referring to the module

Reply:
Oh wait we're looking at `d.Path` instead of `d.Main`, in that case it is referring to package you're right.

Reply:
`mainPackage` or `upstreamPackage` would be more suitable then

Reply:
Thanks @joao-r-reis .  I kinda like "mainPackage" in this case so I ran with it... if anybody has objections let me know!